packages/test/test-utils/api-report/test-utils.api.md

```diff
@@ -223,12 +224,13 @@ export interface ITestObjectProvider {
     ensureSynchronized(timeoutDuration?: number): Promise<void>;
     loadContainer(entryPoint: fluidEntryPoint, loaderProps?: Partial<ILoaderProps>, requestHeader?: IRequestHeader): Promise<IContainer>;
     loadTestContainer(testContainerConfig?: ITestContainerConfig, requestHeader?: IRequestHeader): Promise<IContainer>;
-    logger: EventAndErrorTrackingLogger | undefined;
+    logger: ITelemetryBaseLogger;
     makeTestContainer(testContainerConfig?: ITestContainerConfig): Promise<IContainer>;
     makeTestLoader(testContainerConfig?: ITestContainerConfig): IHostLoader;
     opProcessingController: IOpProcessingController;
     reset(): void;
     resetLoaderContainerTracker(syncSummarizerClients?: boolean): any;
+    tracker: EventAndErrorTrackingLogger;
```

Comment:
IMO, it's worth it to restrict the type of this in the public API to just be the "event/error tracking" part of `EventAndErrorTrackingLogger` (and not the logging part). That prevents someone from accidentally calling `.send` on the tracker. Also makes it more clear how to document this field (which currently has duplicated doc to logger)